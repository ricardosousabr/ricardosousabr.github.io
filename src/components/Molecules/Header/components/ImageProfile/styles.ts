import styled from "styled-components";

export const ImageProfileStyle = styled.div`
  padding: 70px 0;
  display: flex;
  align-items: center;
  justify-content: center;
`