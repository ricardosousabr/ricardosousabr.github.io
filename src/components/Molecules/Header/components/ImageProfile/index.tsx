import Image from "next/image";
import { ImageProfileStyle } from "./styles";

export default function ImageProfile() {
   return (
    <ImageProfileStyle>
         <Image width={400} height={500} src="/img/Profile.png" alt="" />
    </ImageProfileStyle>
   )
}