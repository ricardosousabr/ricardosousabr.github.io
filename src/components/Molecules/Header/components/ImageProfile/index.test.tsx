import renderer from "react-test-renderer";
import ImageProfile from "./index";

test("Link renders correctly", () => {
  const tree = renderer.create(<ImageProfile />).toJSON();
  expect(tree).toMatchSnapshot();
});
