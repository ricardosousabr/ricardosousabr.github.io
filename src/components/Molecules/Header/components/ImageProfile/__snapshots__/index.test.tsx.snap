// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
<div
  className="sc-bdnyFh bXaykT"
>
  <img
    alt=""
    data-nimg="1"
    decoding="async"
    height={500}
    loading="lazy"
    onError={[Function]}
    onLoad={[Function]}
    src="/_next/image?url=%2Fimg%2FProfile.png&w=828&q=75"
    srcSet="/_next/image?url=%2Fimg%2FProfile.png&w=640&q=75 1x, /_next/image?url=%2Fimg%2FProfile.png&w=828&q=75 2x"
    style={
      {
        "color": "transparent",
      }
    }
    width={400}
  />
</div>
`;
